packages/plugin-board/src/board.js

```diff
@@ -57,6 +57,47 @@ const Board = SparkPlugin.extend({
   },
 
   /**
+   * Adds a snapshot image of the board
+   *
+   * @param {Conversation} conversation - the current conversation that the board belongs
+   * @param {Board~Channel} channel
+   * @param {File} image
+   * @returns {Promise<Board~Channel>}
+   */
+  addSnapshotImage(conversation, channel, image) {
```

Comment:
I can make the change in the new PR (ACL changes https://github.com/ciscospark/spark-js-sdk/pull/357)